plugins/inputs/beat/beat.go

```diff
@@ -0,0 +1,278 @@
+package beat
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	jsonparser "github.com/influxdata/telegraf/plugins/parsers/json"
+)
+
+const sampleConfig = `
+  ## An URL from which to read Beat-formatted JSON
+  ## Default is "http://127.0.0.1:5066".
+  url = "http://127.0.0.1:5066"
+
+  ## Enable collection of the Beat stats
+  collect_beat_stats = true
+
+  ## Enable the collection if Libbeat stats
+  collect_libbeat_stats = true
+
+  ## Enable the collection of OS level stats
+  collect_system_stats = false
+
+  ## Enable the collection of Filebeat stats
+  collect_filebeat_stats = true
+
+  ## HTTP method
+  # method = "GET"
+
+  ## Optional HTTP headers
+  # headers = {"X-Special-Header" = "Special-Value"}
+
+  ## Override HTTP "Host" header
+  # host_header = "logstash.example.com"
+
+  ## Timeout for HTTP requests
+  timeout = "5s"
```

Comment:
Same. Please comment this option with the (good) default to not confuse new users.

Reply:
Will fix